slither-audit.txt

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
We should have the log files ignored in .gitignore and not check them in

Reply:
Ahh okay, I'll implement it